src/main/kotlin/io/snyk/plugin/SnykPostStartupActivity.kt

```diff
@@ -43,6 +49,17 @@ class SnykPostStartupActivity : StartupActivity.DumbAware {
             SnykBalloonNotifications.showFeedbackRequest(project)
             settings.lastTimeFeedbackRequestShown = Date.from(Instant.now())
         }
-    }
 
+        val userToken = settings.token ?: ""
+        if (userToken.isNotBlank()) {
+            LOG.info("Loading variants for all amplitude experiments")
+            val publicUserId = service<SnykApiService>().userId ?: ""
+            val experimentUser = ExperimentUser(publicUserId)
+            service<AmplitudeExperimentService>().fetch(experimentUser)
```

Comment:
inline experimentUser and publicUserId ?

Reply:
As agreed it's from their SDK